### PR TITLE
[JSX] Media - Refactor and Clean-Up

### PR DESCRIPTION
**Pull request by HenriRabalais**
### Summary of changes
This PR refactors the Media jsx file to meet ES6 conventions and include the formatColumns function, rather than have it in a separate file.

### Testing
- Go to the Media module and ensure that everything is functioning.

### CODE REVIEW

**Comment by HenriRabalais** [comment]
@davidblader @zaliqarosli @kongtiaowang please review/test.

**Comment by zaliqarosli** [comment]
simiar PR referenced above ^ but on major

File: modules/media/jsx/mediaIndex.js [hunk 1]
```diff
@@ -90,18 +154,23 @@ class MediaIndex extends React.Component {
...
+              type="reset"
+              onUserInput={this.resetFilters}
+            />
           </FilterForm>
           <StaticDataTable
-            Data={this.state.Data.Data}
-            Headers={this.state.Data.Headers}
+            Data={this.state.data.Data}
+            Headers={this.state.data.Headers}
+            hiddeHeaders={this.state.hiddenHeaders}
```
Comment (zaliqarosli) [inline modules/media/jsx/mediaIndex.js]: typo for hiddenHeaders

File: modules/media/jsx/mediaIndex.js [hunk 2]
```diff
@@ -90,18 +154,23 @@ class MediaIndex extends React.Component {
...
+              type="reset"
+              onUserInput={this.resetFilters}
+            />
           </FilterForm>
           <StaticDataTable
-            Data={this.state.Data.Data}
-            Headers={this.state.Data.Headers}
+            Data={this.state.data.Data}
+            Headers={this.state.data.Headers}
+            hiddenHeaders={this.state.hiddenHeaders}
```
Comment (zaliqarosli) [inline modules/media/jsx/mediaIndex.js]: i just realized that this.props.hiddenHeaders hasn't been implemented yet on minor

Comment (HenriRabalais) [reply modules/media/jsx/mediaIndex.js]: Nope... hence the frustration of these sort of in-between PRs while waiting for the merge. Can remove them or leave them in the meantime but would really not like to keep going back and forth on them.

**Pull request merged**